src/System.Net.Http/src/System/Net/Http/Managed/AuthenticateAndRedirectHandler.Helpers.cs

```diff
@@ -2,34 +2,56 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Why is this in a separate file rather than just part of the AuthenticateAndRedirectHandler.cs file?

Reply:
To keep organized, the handler file contains only the relevant SendAsync implementation for the handler, constructor and Dispose like other handlers. I can club all into one, if this is confusing.

Reply:
> I can club all into one, if this is confusing.

Thanks. I find it confusing.